src/ansys/fluent/core/services/field_data.py

```diff
@@ -12,6 +13,13 @@
 from ansys.fluent.core.services.error_handler import catch_grpc_error
 from ansys.fluent.core.services.interceptors import TracingInterceptor
 
+
+def override_help_text(func, func_to_be_wrapped):
+    func.__doc__ = "\n" + pydoc.text.document(func_to_be_wrapped)
```

Comment:
Is it possible to set this within _FieldMethod.__init__?

Reply:
@mkundu1 That was what we were trying to do initially, and could not get it to update, as it is read-only. Whether or not such a change can work within the current code, perhaps we could alternatively still streamline this code a bit by adding one more level of indirection.

Reply:
Thank you.